source/_integrations/iotty.markdown

```diff
@@ -41,22 +41,28 @@ US version:
 
 EU version:
 
-- [iotty Smart Switch](https://iotty.uk/collections/frontpage/products/iotty-smart-switch)
-- [iotty Smart Switch (variant)](https://iotty.uk/collections/frontpage/products/iotty-smart-switch?variant=40820222460082)
-- [iotty Plus Interruttore Intelligente](https://iotty.it/collections/frontpage/products/iotty-plus-interruttore-intelligente-per-luci-e-cancelli)
+- [iotty Smart Switch](https://iotty.uk/collections/prodotti-singoli/products/e1-e2-plus-smart-switch-for-lights-and-gates)
+- [iotty Smart Switch (variant)](https://iotty.uk/collections/prodotti-singoli/products/e1-e2-plus-smart-switch-for-lights-and-gates?variant=48626603032911)
+- [iotty Plus Interruttore Intelligente](https://iotty.it/collections/prodotti-singoli/products/i3-plus-interruttore-intelligente-per-luci-e-cancelli)
 
 ### iotty Shutter
 
 - [iotty Smart Shades Switch](https://iotty.uk/collections/frontpage/products/e2s-plus-smart-shades-switch-for-shutters-and-blinds)
 - [iotty Plus Interruttore Intelligente per Tende e Tapparelle](https://iotty.it/collections/prodotti-singoli/products/i3s-plus-interruttore-intelligente-per-tende-e-tapparelle) (currently only available for the Italian market)
 
+### iotty Outlet
+
+- [iotty Smart Outlet - Italy](https://iotty.it/collections/prodotti-singoli/products/oit-plus-presa-intelligente)
+- [iotty Smart Outlet - Germany](https://iotty.fr/collections/prodotti-singoli/products/ofr-plus-prise-intelligente)
+- [iotty Smart Outlet - France](https://iotty.de/collections/prodotti-singoli/products/ode-plus-smarte-steckdose)
```

Comment:
You probably swapped these links.

Reply:
Nice catch, thanks a lot! I have fixed this